### ISSUE TITLE
Failures in native voice notifications can cause crash

### ISSUE THREAD

**Issue by fredjk-gh** [Xjph/ObservatoryCore#73]
Reported by a few folks in the wild, reproduced by Matt G via BioInsights plugin. No error log was produced in Documents. The following error is from Event Viewer:

```
Application: ObservatoryCore.exe
CoreCLR Version: 5.0.721.25508
.NET Version: 5.0.7
Description: The process was terminated due to an unhandled exception.
Exception Info: System.FormatException: 'xml:lang' element not valid.
 ---> System.Globalization.CultureNotFoundException: Culture is not supported. (Parameter 'name')
string is an invalid culture identifier.
   at System.Globalization.CultureInfo..ctor(String name, Boolean useUserOverride)
   at System.Speech.Internal.Synthesis.SsmlParser.ProcessSpeakElement(XmlReader reader, ISsmlParser engine, Object voice)
   --- End of inner exception stack trace ---
   at System.Speech.Internal.Synthesis.SsmlParser.ThrowFormatException(Exception innerException, SRID id, Object[] args)
   at System.Speech.Internal.Synthesis.SsmlParser.ProcessSpeakElement(XmlReader reader, ISsmlParser engine, Object voice)
   at System.Speech.Internal.Synthesis.SsmlParser.Parse(XmlReader reader, ISsmlParser engine, Object voice)
   at System.Speech.Internal.Synthesis.SsmlParser.Parse(String ssml, ISsmlParser engine, Object voice)
   at System.Speech.Internal.Synthesis.VoiceSynthesis.ThreadProc()
--- End of stack trace from previous location ---
   at System.Speech.Internal.Synthesis.VoiceSynthesis.Speak(Prompt prompt)
   at System.Speech.Synthesis.SpeechSynthesizer.Speak(Prompt prompt)
   at Observatory.NativeNotification.NativeVoice.ProcessQueue() in C:\Users\Xjph\Source\Repos\ObservatoryCore\ObservatoryCore\NativeNotification\NativeVoice.cs:line 68
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Observatory.NativeNotification.NativeVoice.ProcessQueueAsync() in C:\Users\Xjph\Source\Repos\ObservatoryCore\ObservatoryCore\NativeNotification\NativeVoice.cs:line 35
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__140_1(Object state)
   at System.Threading.QueueUserWorkItemCallbackDefaultContext.Execute()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
```

Seems we're missing a top-level error handler somewhere.

**Comment by fredjk-gh** [comment]
I think this was fixed by #74 ...

**Comment by Xjph** [comment]
I'll close this and we'll see if it resurfaces then.
